File: 프로그래머스/0/181906. 접두사인지 확인하기/접두사인지 확인하기.cs
using System;
using System.Linq;

public class Solution {
    public int solution(string my_string, string is_prefix) {
        string[] arr = new string[my_string.Length];
        
        for(int i = 0; i < my_string.Length; i++)
        {
            arr[i] = my_string.Substring(0, i);
        }
        
        return arr.Contains(is_prefix) ? 1 : 0;
    }
}

File: 프로그래머스/0/181906. 접두사인지 확인하기/접두사인지 확인하기.cpp
#include <string>
#include <vector>
#include <bits/stdc++.h>

using namespace std;

int solution(string my_string, string is_prefix) {
    vector<string> arr;
    string temp = "";
    for(int i = 0; i < my_string.size(); i++)
    {
        temp.push_back(my_string[i]);
        arr.push_back(temp);
    }
    if(find(arr.begin(), arr.end(), is_prefix) != arr.end()) return 1;
    return 0;
}